Marlin/tests/core/test_macros.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include "../test/unit_tests.h"
 #include <src/core/macros.h>
+#include <vector>
+#include <array>
```

Comment:
These includes aren't needed since GitHub has fixed the issue with their "ubuntu-latest" runner image.